### PR TITLE
[Refine] Custom Ops: add attention logits output

### PR DESCRIPTION
**Pull request by addf400**
add attention logits output

### CODE REVIEW

**Comment by 0xWJ** [comment]
Custom Ops Module was added by Yi for some reason, but the interface looks still unstable. I am worried about that a little bit. I hope we can have some degree of interface stability in near future (at least have backward compatibility)

**Review by 0xWJ — APPROVED** [review]
